techmatters/flex-plugins
# Instagram channel registration and UI implementation of IG assets

Primary reviewer: @GPaoloni 

## Description
- Set up and registered custom Instagram Channel
- Added Instagram logo and color in All Tasks and Timeline components
- Added queue status functionality to Contacts waiting widget for incoming instagram messages
- Changes made according to [this](https://marvelapp.com/prototype/6e73aca/screen/85221384) design document. Note a change is made to use solid color instead of gradient for the icon at Timeline component after discussion with Jana

### Checklist
- [N/A] Strings are localized

### Related Issues: [CHI-1057](https://bugs.benetech.org/browse/CHI-1057) and [CHI-1058](https://bugs.benetech.org/browse/CHI-1058)

### Verification steps
- Note the four additions/changes to check for Instagram channel UI integration: Instagram contact window box, Instagram icon at All Tasks and Timeline components, instagram color at chat window start. 
![Instagram UI changes](https://user-images.githubusercontent.com/42842977/155612534-e4e580b5-a7aa-41fd-9b1a-823362d80fb2.png)
